File: ClientApp/src/app/objects/connection.ts
import { Character } from "./character";

export class Connection {
    id: string;
    from: string;
    to: string;
    title: string;

    constructor(
        id: string,
        from: string,
        to: string,
        title: string
    ) {
        this.id = id;
        this.from = from;
        this.to = to;
        this.title = title;
    }
}

File: TreeWikiApi/DAL/Entities/Connection.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DAL.Entities
{
    public class Connection
    {
        public Guid Id { get; set; }

        public string Title { get; set; }

        public Guid From { get; set; }
        
        public Guid To { get; set; }
        
        public Guid WorkId { get; set; }
    }
}
